aiohttp/connector.py

```diff
@@ -814,6 +814,7 @@ def clear_dns_cache(
     async def _resolve_host(
         self, host: str, port: int, traces: Optional[List["Trace"]] = None
     ) -> List[Dict[str, Any]]:
+        """Resolves host and returns list of addresses."""
```

Comment:
PEP 257 requires a sentence in imperative mood, in docstring title lines: https://peps.python.org/pep-0257/#one-line-docstrings

Reply:
2367798d26f4d8928e618976f8d231e2160ce16b